### DISCUSSION TITLE
Any idea when the next version will be finished?

### DISCUSSION THREAD

**Discussion by TheJaysH** [Blazor-Diagrams/Blazor.Diagrams#306] [General]
I'm still using the beta, and just wondering when it will go GA?

**Comment by GeorgeS2019** [comment]
Diagrams (3.0.0-beta.5) - 2022-11-23
https://github.com/Blazor-Diagrams/Blazor.Diagrams/tree/develop

https://github.com/Blazor-Diagrams/Blazor.Diagrams/discussions/271

https://github.com/Blazor-Diagrams/Blazor.Diagrams/releases/tag/3.0.0-beta.5

**Comment by zHaytam** [comment]
Hello.
Still struggling with some IRL issues, the documentation website is like 50% done and the beta version is still usable unless new bugs are reported.

Sorry for the huge delay

**Reply by GeorgeS2019** [reply]
Thx for excellent work, for .NET project will be abandoning React but adopt Blazor Diagram, get ready for a faster bug iteration

**Reply by jeharris-eci** [reply]
Not to worry, hope all is well. keep on keeping on!
